addons/sale_crm/tests/test_cron_assign_leads.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.crm.tests.test_crm_lead_assignment import TestLeadAssignCommon
+from odoo.tests.common import tagged
+
+
+@tagged('lead_assign')
+class TestSalesLeadAssign(TestLeadAssignCommon):
+    def test_cron_assign_leads(self):
```

Comment:
Maybe add a docstring here too like `"""Test leads assignment with sale orders from different companies are not merged."""`